.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please exclude this file from git, using `.gitignore` file. You can search more about that.

Reply:
Please also exclude other 6 files in `__pycache__` folder. Generally, do not commit the files that do not have meaningful code, especially private configuration files or cache files.

Reply:
Ignore the db file too (`mealplan.db`)